releaseNotes/Microsoft.Featuremanagement.md

```diff
@@ -2,6 +2,17 @@
 
 [Source code][source_code] | [Package (NuGet)][package] | [Samples][samples] | [Product documentation][docs]
 
+## 3.3.0 - May 8, 2024
+
+### Enhancements
+
+* Added a `Recurrance` option to the `TimeWindow` filter. This allows a recurring period when the filter will be on. Now you can enable the flag every day after 5 PM or perhaps on Fridays from noon to 6 PM. ([#266](https://github.com/microsoft/FeatureManagement-Dotnet/pull/266)). See more details [here](https://github.com/microsoft/FeatureManagement-Dotnet?tab=readme-ov-file#recurrence-pattern).
+* Adjusted the built-in filters to optionally require a `LoggerFactory`. ([#386](https://github.com/microsoft/FeatureManagement-Dotnet/pull/386)).
```

Comment:
This is confusing me. Is it that we no longer require logger factory?

Reply:
optionally required sounds like an oxymoron.

Reply:
Yes. The built-in filters will default to null if no LoggerFactory is available. Do you prefer:

Made `LoggerFactory` an optional dependency for built-in filters

Reply:
Reasonable, but dependency is vague. How about 

A `LoggerFactory` is no longer a required when constructing built-in filters.

Reply:
Adjusted to

> A `LoggerFactory` is no longer required when constructing built-in filters.